config/settings.defaults.coffee

```diff
@@ -16,6 +16,15 @@ httpAuthUsers[httpAuthUser] = httpAuthPass
 sessionSecret = "secret-please-change"
 
 module.exports = settings =
+
+
+	# Overleaf overide settings
+	# -------------------------
+	# These should be named the same as corisponding ShareLaTeX setting when possible.
+	overleaf: 
+
+		cookieDomain:".overleaf.local"
+
```

Comment:
we could have a separate overleaf config file still with overides that are read when the site is in overleaf mode